charmcraft/commands/store/client.py

```diff
@@ -47,6 +47,34 @@ def build_user_agent():
     )
 
 
+class AnonymizedClient(craft_store.http_client.HTTPClient):
```

Comment:
Possibly `AnonymousClient` is a better name; "Anonymized" makes me think that the client strips identifying info from the requests somehow, which isn't the case

Reply:
OK